test/journeys/lib/test-helpers/basic.js

```diff
@@ -0,0 +1,28 @@
+export const elements = {
```

Comment:
I don't think we are using these in the recents widget. `oneOnOne` and `space` are using the same `space-widget`, just in different modes.